internal/pkg/version/gen/generate-version.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// This program generates the version and version hash. It is invoked by running `go generate` from the root command
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	// "time"
+
+	"text/template"
+)
+
+const versionTemplate = `// Code generated by go generate; DO NOT EDIT.
+// This file was generated by 🤖
+package version
+
+const Version = "{{.Version}}"
+var GitHash string
+var Platform string
+`
```

Comment:
Can this `version` package simply exist without the need to "generate" it and rely on the `ldflags` process to fill in values? It seems like the whole `generate-version` mini-application could go away entirely?